fortune/Dockerfile.dc

```diff
@@ -1,19 +0,0 @@
-FROM golang:1.10
-
-RUN apt update && apt install -y unzip time make
-
-ENV PROTOC_VERSION 3.5.1
-
-RUN wget https://github.com/google/protobuf/releases/download/v${PROTOC_VERSION}/protoc-${PROTOC_VERSION}-linux-x86_64.zip && \
-  unzip protoc-${PROTOC_VERSION}-linux-x86_64.zip -d protoc && \
-  mv protoc/bin/protoc /usr/bin/protoc
-
-RUN go get github.com/golang/protobuf/protoc-gen-go
-
-ADD . /go/src/github.com/tilt-dev/servantes/fortune
-RUN cd /go/src/github.com/tilt-dev/servantes/fortune && make proto
-RUN go install github.com/tilt-dev/servantes/fortune
-
-ENV TEMPLATE_DIR /go/src/github.com/tilt-dev/servantes/fortune/web/templates
-
-ENTRYPOINT /go/bin/fortune
```

Comment:
Not sure if these were separate dockerfiles for a reason, to test the `dockerfile=` arg maybe? I converted them to symlinks, we could also just delete the dockerfile arg and the *.dc files/links.